api/evidence/evidence_store.proto

```diff
@@ -80,11 +80,19 @@ message StoreEvidencesResponse {
 }
 
 message ListEvidencesRequest {
+  // page_size: 0 = default, > 0 = set value
   int32 page_size = 1;
   string page_token = 2;
   string order_by = 3;
   bool asc = 4;
+  Filter filter = 5;
```

Comment:
I am not sure if we should make this optional @oxisto @anatheka ?

Reply:
I'm not sure if we should set the Filter or the properties of the filter optional. Does one or the other have any implications?

Reply:
I would think to apply optional to Filter as well as for the fields in it. Because we can choose if we want to apply any filter or not (making `Filter` optional) and then we choose which exact filter we want to apply (making `Filter`s fieleds opional. I guess, at least, these fields have to be optional anyway).

@luis-gasparschroeder Please do not resolve conversations where we not discussed the matter conclusively :)

Reply:
I made the Filter optional